docs/_importers/medium.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: Medium
+prev_section: marley
+link_source: medium
+next_section: mephisto
+permalink: /docs/medium/
+---
+
+To import your posts from [Medium](https://medium.com/), run:
+
+{% highlight bash %}
+$ ruby -r rubygems -e 'require "jekyll-import";
+    JekyllImport::Importers::Medium.run({
+        "username"       => "name",
+        "render_audio"   => "false",
```

Comment:
Same here, is this also meant to be a plain Boolean or a string value?

Reply:
Updated the example